app/mailers/offline_cancellation_mailer.rb

```diff
@@ -0,0 +1,20 @@
+class OfflineCancellationMailer < BaseMailer
+
+  attr_reader :reservation
+
+  def send_notification(reservation)
+    @reservation = reservation
+    @instrument = reservation.product
+    mail(to: reservation.user.email, subject: subject)
+  end
+
+  private
+
+  def subject
+    text(
+      "views.notifier.offline_cancellation_notification.subject",
```

Comment:
I'm not convinced this locale should live in`notifier`. If we pull the text out of the templates, then it probably makes sense to give it its own file. I'm still not sure what an optimal file organization is for locales, but I think I'm liking the one file per feature (so it could include views, controller, mailers, and possibly models in the same file).